runtime/parachains/src/configuration.rs

```diff
@@ -278,6 +277,28 @@ pub trait WeightInfo {
 	fn set_hrmp_open_request_ttl() -> Weight;
 }
 
+pub struct TestWeightInfo;
+impl WeightInfo for TestWeightInfo {
+	fn set_config_with_block_number() -> Weight {
+		frame_system::limits::BlockWeights::default().max_block
+	}
+	fn set_config_with_u32() -> Weight {
+		frame_system::limits::BlockWeights::default().max_block
```

Comment:
this seems very arbitrary, since this might be "zero" for some other chain. Should just be `Weight::MAX`?